src/Core/src/Platform/iOS/IMauiPlatformView.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.Maui.Platform;
+
+internal interface IMauiPlatformView
```

Comment:
nit: this is very general and I am not 100% certain we won't need this name for a more general thing. Maybe this can be `IMauiMeasurePlatformView` instead? Thoughts?

Reply:
You're right.. some other ideas:
- `IMeasureInvalidatablePlatformView`
- `IMeasureInvalidationController`